modules/lib/shell.nix

```diff
@@ -2,7 +2,11 @@
 
 rec {
   # Produces a Bourne shell like variable export statement.
-  export = n: v: ''export ${n}="${toString v}"'';
+  export = n: v:
```

Comment:
If you change the definition of `NIX_DEBUG_INFO_DIRS` to not be a list then you could also remove this change.

Reply:
You're absolutely right! I wasn't even that confident about this change but with your suggestions above this would be a totally unnecessary breaking-change :)